src/routes/login.tsx

```diff
@@ -111,17 +119,35 @@ export function LoginForm() {
               </div>
               <div className="grid gap-2">
                 {Role.Admin === role ? (
-                  <>
-                    <Label htmlFor="key" className="flex items-center">
+                  <div className="relative">
+                    <Label htmlFor="key" className="flex items-center mb-2">
                       Admin Key (xpriv)
                     </Label>
-                    <Input id="key" value={key} type="text" placeholder="Admin Key" onChange={onChangeKey} />
-                  </>
+                    <Input
```

Comment:
There is a warning:
> [DOM] Password field is not contained in a form

Reply:
It's in a console in chrome developers tools.

Reply:
It does not necessary to be in `from` as in React you can have your own logic to handle this kind of field.

Reply:
**Good Practices:**
- **Semantics and Accessibility**: Using a <form> element where appropriate improves the semantics of your application, making it more accessible to screen readers and assistive technologies. This can also lead to better keyboard navigation and form interaction.
- **Form Control Organization**: In complex applications, organizing inputs inside forms (or even nested forms) can help manage state better, especially if you need to differentiate between various sets of inputs.

Reply:
Make sense. Will add form tags